src/components/Context/tests/LenguageContext.spec.tsx

```diff
@@ -0,0 +1,84 @@
+import React, {
+  Dispatch,
+  ReactNode,
+  SetStateAction,
+  useContext,
+  useState,
+} from 'react';
+import { render, screen, act, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { LenguageContext, initialState } from '../LenguageContext';
+
+const TestComponent = () => {
+  const { language, setLanguage } = useContext(LenguageContext);
+
+  return (
+    <div>
+      <span data-testid="language-value">{language}</span>
+      <button onClick={() => setLanguage('English')}>Set to English</button>
```

Comment:
ISO code of lang (the same will be with json names)"en" https://www.loc.gov/standards/iso639-2/php/code_list.php